src/lsp/process.rs

```diff
@@ -295,12 +295,7 @@ impl LspServerProcess {
             None,
             InitializeParams {
                 process_id: None,
-                root_uri: Some(Url::parse(&format!(
-                    "file://{}",
-                    self.current_working_directory.display_absolute()
-                ))?),
```

Comment:
Why is this removed?

Reply:
`root_uri` was deprecated in `lsp_types` in favor of `workspace_folders`. With `root_uri` still in the code a warning was omitted about the deprecation.

See: https://docs.rs/lsp-types/0.97.0/lsp_types/struct.InitializeParams.html#structfield.root_uri